packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -98,27 +98,35 @@ const FolderTreeItem = ({
     cx(
       'folder-tree-item',
       { selected: isSelected },
-      { folder: isFolder },
       theme.className,
     ),
   );
 
+  const handleRadioButtonClick = (event) => {
+    // Stop click propagation to prevent triggering expand/collapse when selecting folders
+    event.stopPropagation();
+
+    onClick();
```

Comment:
Since onClick is not required, we either need to check here or when the event handler is defined to prevent an exception.

Reply:
Good catch, [deebcdc](https://github.com/cerner/terra-framework/pull/1937/commits/deebcdce1dfbe6dbdd44fd9e553aa3076b58383a)